### DISCUSSION TITLE
How to obtain SeuratObject from ArchRProject?

### DISCUSSION THREAD

**Discussion by YayanFeng55** [GreenleafLab/ArchR#1975] [Questions / Documentation]
Dear all,

Is there any function to convert an ArchRProject to SeuratObject? 

Thanks
Yayan

**Comment by ronghai-li** [comment]
https://github.com/swaruplabUCI/ArchRtoSignac
this R package is work for you?

**Reply by YayanFeng55** [reply]
I will try this package. Thanks a lot
